lib/compiler/aro_translate_c.zig

```diff
@@ -1622,6 +1625,33 @@ test "Macro matching" {
     try helper.checkMacro(allocator, pattern_list, "IGNORE_ME(X) ((volatile const void)(X))", "DISCARD");
 }
 
+/// Renders errors and fatal errors + associated notes (e.g. "expanded from here"); does not render warnings or associated notes
+/// Terminates with exit code 1
+fn renderErrorsAndExit(comp: *aro.Compilation) noreturn {
+    defer std.process.exit(1);
```

Comment:
Is it safe/valid to have a `defer` which runs after control flow reaches the end of a `noreturn` function or is this too clever?

Reply:
`defer` statements run at the end of the current block, so afaiu the language has to allow it if you added another block scope encompassing the entire function body (that block would be `void`).
I'm not sure how much the compiler would gain by requiring this extra step of users, so I wouldn't see a point in disallowing it as-is.